ppoprf/src/ppoprf.rs

```diff
@@ -84,12 +85,33 @@ impl ProofDLEQ {
 pub type ServerPublicKey = Vec<RistrettoPoint>;
 
 // The wrapper for PPOPRF evaluations (similar to standard OPRFs)
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize)]
 pub struct Evaluation {
+    #[serde(serialize_with = "ristretto_serialize")]
     output: CompressedRistretto,
     proof: Option<ProofDLEQ>,
 }
 
+#[derive(Deserialize)]
+pub struct Points(
```

Comment:
There's only one point here, so the struct name should probably be singular.

Reply:
gotcha!